demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -599,36 +632,39 @@ static bool getTemporaryCredentials( TransportInterface_t * transportInterface,
 
         if( jsonStatus != JSONSuccess )
         {
-            LogError( ( "Failed to parse temporary IOT credentials. " ) );
+            LogDebug( ( "AWS IoT credential provider response: %s.",
+                        response.bufferLen, response.pBuffer ) );
```

Comment:
Please make sure that the debug level logging code builds fine
```suggestion
            LogDebug( ( "AWS IoT credential provider response: %.*s.",
                        response.bufferLen, response.pBuffer ) );
```

Reply:
This log message can exist out of the `if( jsonStatus != JSONSuccess )` block so that it is logged every time